doc/format.md

```diff
@@ -358,11 +361,18 @@ This should not be a stack offset.
 The parameter is a number; if prefixed with `0x` then in hex format, otherwise, decimal format. Negative offsets are supported.
 An offset can be followed by an optional description.
 
+If the number is only relevant for a particular architecture, then you can use one of the architecture flavors: `number/x32` or `number/x64`.

```

Comment:
fix:
If the offset is only relevant for a particular architecture, then you can use one of the architecture flavors: `offset/x32` or `offset/x64`.